test/hotspot/jtreg/compiler/vectorization/TestRotateByteAndShortVector.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, Loongson Technology. All rights reserved.
```

Comment:
The copyright line isn't the same with the one in `compiler/compilercontrol/CompilationModeHighOnlyTest.java`.

Reply:
I hava changed.